crates/esthri/src/ops/sync.rs

```diff
@@ -607,17 +607,78 @@ where
         }
     }
 
-    Ok(())
+    if delete {
+        sync_delete_across(
+            s3,
+            source_bucket,
+            source_prefix,
+            dest_bucket,
+            destination_key,
+            filters,
+            Config::global().concurrent_sync_tasks(),
```

Comment:
Should move this into `sync_delete_across` since it's a constant anyway

Reply:
Done